### PR TITLE
fix sidebar overflowing property

### PR DESCRIPTION
**Pull request by tvpeter**
- fix sidebar layout

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
-import * as React from "react";
-import { Parser } from "html-to-react";
-import AppSearchAPIConnector from "@elastic/search-ui-app-search-connector";
+import * as React from 'react';
+import { Parser } from 'html-to-react';
+import AppSearchAPIConnector from '@elastic/search-ui-app-search-connector';
```
Comment (aureleoules) [inline src/App.js:4]: Please don't format the code as it makes reviews difficult.

Comment (tvpeter) [reply src/App.js:4]: noted

**Comment by aureleoules** [comment]
ACK 20d65e10338df82a2e6962c51954e1a94c94ded7 - makes the site more responsive

**Comment by adamjonas** [comment]
rebased ^^

File: src/custom.css [hunk 2]
```diff
@@ -14,16 +14,17 @@
 }
 
 .btc-search .sui-layout-sidebar{
-    width: 30%;
+    overflow-x: hidden;
```
Comment (adamjonas) [inline src/custom.css:17]: I'm not really a fan of this approach. I'd prefer I think to use break points at 800px to make sure it uses the full screen on mobile

**Pull request closed without merging**